docs/explanation.md

```diff
@@ -0,0 +1,253 @@
+
+```python exec="true" idprefix=""
+from ssvc.decision_points.system_exposure import LATEST
+from ssvc.doc_helpers import example_block
+
+print(example_block(LATEST, include_json=False))
+```
+
+#### Automability
+
+*Automability* is the same question as contained in the Supplier's Utility decision point.
+For easy reference, *Automability* is a simple yes or no, can the attacker reliably automate all four steps (reconnaissance, weaponization, delivery, exploitation) of the kill chain? If the vulnerability allows remote code execution or command injection, the expected response should be 'yes.'
+
+#### Human Impact
+
+*Human Impact* is similar to the Supplier's decision point *Public Safety Impact* in that it factors *Safety Impact*, but it differs in that it is a compound decision point to also consider *Mission Impact*.
+
+For easy reference, the information on *Safety Impact* is again provided here:
+
+*Safety Impact* is expansive to include impacts of physical harm, operator resiliency, system resiliency, environment, financial, and psychological harm. We used IEC 61508 to guide determinations of Negligible, Marginal, Critical, and Catastrophic impact on the aforementioned types of harm. Only one type of harm needs to qualify per category. Use the highest qualifying category to determine safety impact.
+
+```python exec="true" idprefix=""
+from ssvc.decision_points.safety_impact import LATEST
+from ssvc.doc_helpers import example_block
+
+print(example_block(LATEST, include_json=False))
+```
+
+*Mission Impact* is the impact on the Mission Essential Functions of the organization. A **mission essential function (MEF)** is a function “directly related to accomplishing the organization’s mission as set forth in its statutory or executive charter” [@FCD2_2017, page A-1].
+
+```python exec="true" idprefix=""
+from ssvc.decision_points.mission_impact import LATEST
+from ssvc.doc_helpers import example_block
+
+print(example_block(LATEST, include_json=False))
+```
+
+We define the following combinatorics of *Safety Impact* and *Mission Impact* values to determine *Human Impact* values:
+
+```python exec="true" idprefix=""
+from ssvc.decision_points.human_impact import LATEST
+from ssvc.doc_helpers import example_block
+
+print(example_block(LATEST, include_json=False))
+```
+
+### Coordinator decision points for Triage
+
+<embed src="../pdf/ssvc_2_coord-triage.pdf" alt="Coordination Triage Tree" type="application/pdf"
+style="width: 100%;"
+height = "700" />
+
+The below decision points are  modeled on [CERT/CC Coordinated Vulnerability Disclosure (CVD) project](howto/coordination_triage_decision/#coordinator-triage-units-of-work), but we encourage Coordinators to define their own reasons to engage in Triage.
+
+Generally speaking, we believe in Supplier agency, and is why the first two questions are about damage control.
```

Comment:
oof. No, that's not right at all.

We're seeking to give priority to cases where our help is actually needed. We don't have the capacity to help everyone with everything. So if the report is already public, there's not much further we can do, except in special circumstances. If the supplier has not been contacted, we'll usually decline in single-supplier cases because we expect the reporter to have taken some effort on their own before coming to us for help.